modules/backend/lang/lv/lang.php

```diff
@@ -191,272 +287,365 @@
-        'empty_import_columns_error' => 'Lūdzu, norādiet kādas kolonnas ir jāimportē.',
-        'match_some_column_error' => 'Lūdzu, vispirms saskanējiet kolonnas.',
-        'required_match_column_error' => 'Lūdzu, norādiet attiecīgu datubāzes lauku obligātam laukam :label.',
-        'empty_export_columns_error' => 'Lūdzu, norādiet kādas kolonnas ir jāeksportē.',
-        'behavior_missing_uselist_error' => 'Jums ir jāisteno kontroliera uzvedība ListController Jūsu kontrolierī un jāiestata "useList: true" opcija Importa un Eksporta konfigurācijā.',
+        'empty_error' => 'Eksporta procesam netika padoti dati',
+        'empty_import_columns_error' => 'Lūdzu, norādiet kuras kolonnas jāimportē.',
+        'match_some_column_error' => 'Lūdzu, vispirms sasaistiet kolonnas.',
+        'required_match_column_error' => 'Lūdzu, norādiet sasaisti obligātajam laukam :label.',
+        'empty_export_columns_error' => 'Lūdzu, norādiet kuras kolonnas ir jāeksportē.',
+        'behavior_missing_uselist_error' => 'Jums ir jāievieš ListController kontroliera uzvedība ar ieslēgtu "useList" parametru.',
         'missing_model_class_error' => 'Lūdzu, norādiet modelClass īpašību priekš :type',
         'missing_column_id_error' => 'Trūkstošs kolonnas identifikators',
         'unknown_column_error' => 'Nezināma kolonna',
-        'encoding_not_supported_error' => 'Izvēlētā faila kodējums nav atpazīts. Lūdzu, izvēlieties Pielāgota formāta opciju ar pareizu kodējumu lai Jūsu fails tiktu ieimportēts.',
+        'encoding_not_supported_error' => 'Izvēlētā faila kodējums nav atpazīts. Lūdzu, izvēlieties pielāgota faila formāta iespēju ar atbilstošu kodējumu, lai fails tiktu veiksmīgi importēts.',
         'encoding_format' => 'Faila kodējums',
+        'encodings' => [
+            'utf_8' => 'UTF-8',
+            'us_ascii' => 'US-ASCII',
+            'iso_8859_1' => 'ISO-8859-1 (Latīņu-1, Rietumeiropas)',
+            'iso_8859_2' => 'ISO-8859-2 (Latīņu-2, Centrāleiropas)',
+            'iso_8859_3' => 'ISO-8859-3 (Latīņu-3, Dienvideiropas)',
+            'iso_8859_4' => 'ISO-8859-4 (Latīņu-4, Ziemeļeiropas)',
+            'iso_8859_5' => 'ISO-8859-5 (Latīņu, Kirilica)',
+            'iso_8859_6' => 'ISO-8859-6 (Latīņu, Arābu)',
+            'iso_8859_7' => 'ISO-8859-7 (Latīņu, Grieķu)',
+            'iso_8859_8' => 'ISO-8859-8 (Latīņu, Ebreju)',
+            'iso_8859_0' => 'ISO-8859-9 (Latīņu-5, Turku)',
+            'iso_8859_10' => 'ISO-8859-10 (Latīņu-6, Ziemeļvalstu)',
+            'iso_8859_11' => 'ISO-8859-11 (Latīņu, Taju)',
+            'iso_8859_13' => 'ISO-8859-13 (Latīņu-7, Baltijas reģiona)',
+            'iso_8859_14' => 'ISO-8859-14 (Latīņu-8, Ķeltu)',
+            'iso_8859_15' => 'ISO-8859-15 (Latīņu-9, Rietumeiropas revīzija ar eiro zīmi)',
+            'windows_1250' => 'Windows-1250 (CP1250, Centrālās un Austrumeiropas)',
+            'windows_1251' => 'Windows-1251 (CP1251)',
+            'windows_1252' => 'Windows-1252 (CP1252)',
```

Comment:
Are these encoding names typically translated in other references or would end users be used to seeing them in English?

Reply:
Yes, this kind of encoding translations can be found in other software (for example, MS Word etc.). I think in general most people still refers to encoding code - ISO-8859-13, Windows-1250 etc.  - and the description part is used mostly as the last option to decide, also, in my opinion, English and Latvian versions doesn't differ so much so users would be able to backtrack translation to English if needed.